src/commands/verify.command.ts

```diff
@@ -0,0 +1,37 @@
+import { Command, CommandMessage, Guard, Description } from "@typeit/discord";
+import { NotBot } from "../guards/NotABot.guard";
+import { Logger } from "../services/logger.service";
+import { ID } from "../enum/id.enum";
+import { HACKATHON } from "../enum/hackathon.enum"
+import { TextChannel } from "discord.js";
+
+interface verifyIn {
+    verifyArgsString: string;
+}
+export abstract class Verify {
+
+    logger = Logger.prototype.getInstance();
+
+    @Command("verify :verifyArgsString")
+    @Description("Verifies the user if the type !verify HackathonName")
+    @Guard(NotBot)
+    async verify(command: CommandMessage<verifyIn>): Promise<void> {
+        const { verifyArgsString } = command.args;
+        this.logger.info("Sending verify");
+
+        const channel = command.channel as TextChannel;
+
+        channel.bulkDelete(99).then(() => { command.channel.send("Please type in !verify and the name of the Hackathon") });
```

Comment:
Attendee instead of hackathon? Also this should probably get from Airtable in the future (but not right now)